react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { change: false };
+    this.TopicList = this.TopicList.bind(this);
+    this.handleClick = this.handleClick.bind(this);
+    this.ListItem = this.ListItem.bind(this);
```

Comment:
You only need to bind your click handler. Be careful not to just blindly bind everything.